### PR TITLE
Saparate packages

### PR DESCRIPTION
**Pull request by yairtawil**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,10 @@
     "@angular/platform-browser": "^2.4.0",
     "@angular/platform-browser-dynamic": "^2.4.0",
     "@angular/router": "^3.4.0",
+    "cases": "file:src\\packages\\cases",
+    "core": "file:src\\packages\\core",
     "core-js": "^2.4.1",
+    "menu": "file:src\\packages\\menu",
     "rxjs": "^5.1.0",
     "zone.js": "^0.7.6"
   },
```
Comment (yairtawil) [inline package.json]: `package.json` shuold inlclude all modules:
  
- Filters
- DataLayers
- Tools
- Algorithms
- Settings

File: src/app/menu-items/menu-items.module.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import {NgModule} from '@angular/core';
```
Comment (osadan) [inline src/app/menu-items/menu-items.module.ts]: looks great 🚀

File: src/common.scss [hunk 3]
```diff
@@ -1,5 +1,3 @@
-$base_color_of_text: black;
-$base_color_of_text1: black;
-$base_color_of_text2: black;
-$base_color_of_text3: blue;
-$base_size_of_text: 20rem;
+$map-color:#84c7d2;
```
Comment (osadan) [inline src/common.scss]: better use css variables https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables

File: src/packages/algorithms/package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "algorithms",
+  "name": "@ansyn/algorithms",
```
Comment (osadan) [inline src/packages/algorithms/package.json]: great . 
we are on the right way . 🎢

File: src/packages/algorithms/package.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@ansyn/algorithms",
+  "name": "algorithms",
   "version": "1.0.0",
```
Comment (osadan) [inline src/packages/algorithms/package.json]: Wao. looks like you are chasing your tail 🚯

File: src/app/app.component.scss [hunk 6]
```diff
@@ -1,11 +1,5 @@
 @import "../common.scss";
 
-
-button{
```
Comment (osadan) [inline src/app/app.component.scss]: I think it was a very good idea to remove this button. 
good job 🥇

File: src/app/app.component.scss [hunk 7]
```diff
@@ -1,11 +1,5 @@
...
-  position: absolute;
-  right: 0;
-  top:0;
```
Comment (osadan) [inline src/app/app.component.scss]: top:0 that's ins-pairing 💠

File: package.json [hunk 8]
```diff
@@ -3,6 +3,8 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
+    "postinstall": "npm link ./src/packages/algorithms ./src/packages/cases ./src/packages/core ./src/packages/data-layers ./src/packages/filters ./src/packages/menu ./src/packages/menu-items ./src/packages/settings ./src/packages/tools",
```
Comment (osadan) [inline package.json]: good job. 
very interesting concept you found out there 💯

File: package.json [hunk 9]
```diff
@@ -20,15 +22,7 @@
     "@angular/platform-browser": "^2.4.0",
     "@angular/platform-browser-dynamic": "^2.4.0",
     "@angular/router": "^3.4.0",
-    "core": "file:src\\packages\\core",
```
Comment (osadan) [inline package.json]: BARUCH SHEPETARANO

File: src/app/app.component.ts [hunk 10]
```diff
@@ -1,5 +1,5 @@
 import {Component} from '@angular/core';
-import {StoreService} from "core";
+import {StoreService} from "../packages/core/store.service";
```
Comment (osadan) [inline src/app/app.component.ts]: need to be imported from @ansyn/core

File: src/app/app.module.ts [hunk 11]
```diff
@@ -3,9 +3,9 @@ import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
 import { AppComponent } from './app.component';
-import {MenuItemsModule} from "./menu-items/menu-items.module";
-import {CoreModule} from "core";
-import {MenuModule} from "menu/menu.module";
+import {MenuItemsModule} from "@ansyn/menu-items";
+import {CoreModule} from "@ansyn/core";
+import {MenuModule} from "@ansyn/menu";
```
Comment (osadan) [inline src/app/app.module.ts]: good job

File: src/packages/algorithms/package.json [hunk 12]
```diff
@@ -8,5 +8,10 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "peerDependencies": {
+    "@angular/common": "~2.4.0",
+    "@angular/core": "^2.4.0",
+    "@ansyn/core": "file:///C:\\Users\\user\\Desktop\\Development\\ansyn\\src\\packages\\core"
```
Comment (osadan) [inline src/packages/algorithms/package.json]: what is the file mean? 
shouldn't we require that with npm link

Comment (osadan) [inline src/packages/algorithms/package.json]: change to relative path

File: src/packages/cases/package.json [hunk 13]
```diff
@@ -9,9 +9,9 @@
   "keywords": [],
   "author": "",
   "license": "ISC",
-  "dependencies": {
+  "peerDependencies": {
     "@angular/common": "~2.4.0",
     "@angular/core": "^2.4.0",
-    "core": "file:///C:\\Users\\user\\Desktop\\Development\\ansyn\\src\\packages\\core"
+    "@ansyn/core": "file:///C:\\Users\\user\\Desktop\\Development\\ansyn\\src\\packages\\core"
```
Comment (osadan) [inline src/packages/cases/package.json]: same as above

File: src/packages/data-layers/package.json [hunk 14]
```diff
@@ -8,5 +8,10 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "peerDependencies": {
+    "@angular/common": "~2.4.0",
+    "@angular/core": "^2.4.0",
+    "@ansyn/core": "file:///C:\\Users\\user\\Desktop\\Development\\ansyn\\src\\packages\\core"
```
Comment (osadan) [inline src/packages/data-layers/package.json]: same

File: src/packages/filters/package.json [hunk 15]
```diff
@@ -8,5 +8,10 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "peerDependencies": {
+    "@angular/common": "~2.4.0",
+    "@angular/core": "^2.4.0",
+    "@ansyn/core": "file:///C:\\Users\\user\\Desktop\\Development\\ansyn\\src\\packages\\core"
```
Comment (osadan) [inline src/packages/filters/package.json]: same

File: src/packages/menu-items/package.json [hunk 16]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@ansyn/menu-items",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "~2.4.0",
+    "@angular/core": "^2.4.0",
+    "@ansyn/core": "file:///C:\\Users\\user\\Desktop\\Development\\ansyn\\src\\packages\\core",
```
Comment (osadan) [inline src/packages/menu-items/package.json]: wa. wa. 🇱🇷

File: src/packages/tools/tools.module.ts [hunk 17]
```diff
@@ -1,7 +1,7 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { ToolsComponent } from './tools/tools.component';
-import {CoreModule, StoreService, MenuItem} from "core";
+import {CoreModule, StoreService, MenuItem} from "@ansyn/core";
```
Comment (osadan) [inline src/packages/tools/tools.module.ts]: good job

**Comment by osadan** [comment]
resolve #10,#14

**Comment by osadan** [comment]
close #10

**Pull request merged**